Mathlib/Tactic/Attr/Register.lean

```diff
@@ -96,3 +96,40 @@ register_simp_attr enat_to_nat_coe
 
 /-- A simp set for the `pnat_to_nat` tactic. -/
 register_simp_attr pnat_to_nat_coe
+
+/-- `mon_tauto` is a simp set to prove about morphisms from some power of `M` to `M`, where `M` is a
+(commutative) monoid object in a (braided) monoidal category.
+
+THIS SIMP SET IS INCOMPATIBLE WITH THE STANDARD SIMP SET.
```

Comment:
```suggestion
This `simp` set is incompatible with the standard simp set.
```

A quick regex search shows that there is basically no allcaps sentences in docstrings, so I’d say it’s better to avoid launching a bad trend.